packages/terra-functional-testing/src/webpack-server/webpack-server.js

```diff
@@ -11,6 +13,7 @@ class WebpackServer {
     this.config = WebpackServer.config(options);
     this.host = host || '0.0.0.0';
     this.port = port || '8080';
+    this.gridUrl = options.gridUrl ? `http://${options.gridUrl}:80/status` : undefined;
```

Comment:
Should we rename `this.gridUrl` to something like `this.gridStatusUrl` or `this.gridHealthUrl` to avoid confusion with the existing `gridUrl` CLI option that is typically `grid314.test.devcernerpowerchart.com`?

Reply:
Changed to `gridStatusUrl` https://github.com/cerner/terra-toolkit/pull/679/commits/900b6e80e6616822114af05e0ef367660049fb53#diff-73928b81969e1fbd0b7fed98d5dc39d99e26fd5d75ace8ef586e3a4b2e25e0ffR15